library/initial-content/src/main/java/org/cru/godtools/init/content/task/Tasks.kt

```diff
@@ -64,21 +66,25 @@ internal class Tasks @Inject constructor(
         }
     }
 
-    fun initSystemLanguages() {
+    suspend fun initSystemLanguages() {
```

Comment:
What does ```suspend``` add to this?

Reply:
`suspend` marks a function as being usable within a coroutine only. This gives it the capability to call other `suspend` functions which could "suspend" execution of the coroutine until a specific state is met.